packages/client/src/declarations/submissionMiddleware.ts

```diff
@@ -169,7 +169,9 @@ export const submissionMiddleware: Middleware<{}, IStoreState> =
           }
         })
         const { isPotentiallyDuplicate, trackingId, compositionId } =
-          response?.data?.createBirthRegistration ?? {}
+          event === Event.Birth
+            ? response?.data?.createBirthRegistration
+            : response?.data?.createDeathRegistration
```

Comment:
```suggestion
          response?.data?.createBirthRegistration ?? response?.data?.createDeathRegistration ?? {}
```
otherwise it would crash in case of marriage event